bigframes/ml/llm.py

```diff
@@ -144,7 +144,7 @@ def _from_bq(
         kwargs: dict = {}
         last_fitting = model.training_runs[-1]["trainingOptions"]
 
-        dummy_text_generator = cls()
+        dummy_text_generator = cls(session=session)
```

Comment:
I don't think this is the issue. It is only a dummy object to retrieve param types. The actual creation is in line 155.

Reply:
This does call the `.ctor` -> `_create_bqml_model` -> `create_bq_connection`?

Reply:
I see. Thanks for figuring out.

Reply:
Actually this whole block is unnecessary and does more than what we want. I'll send another PR to refactor some.